VideoLocker/res/values/strings.xml

```diff
@@ -68,7 +68,7 @@
     <!-- Message shown while waiting for action -->
     <string name="progressText">Please wait....</string>
```

Comment:
@BenjiLee Should we also remove the ellipses in "Please wait"? If we keep them, there should only be 3 dots, not 4.